src/main/java/com/zendesk/maxwell/schema/ddl/SchemaChange.java

```diff
@@ -35,6 +35,8 @@
 		SQL_BLACKLIST.add(Pattern.compile("^ANALYZE\\s+TABLE", Pattern.CASE_INSENSITIVE));
 		SQL_BLACKLIST.add(Pattern.compile("^SET\\s+PASSWORD", Pattern.CASE_INSENSITIVE));
 		SQL_BLACKLIST.add(Pattern.compile("^(CREATE|DROP|RENAME)\\s+USER", Pattern.CASE_INSENSITIVE));
+
+		SQL_BLACKLIST.add(Pattern.compile("^TRUNCATE\\s+TABLE", Pattern.CASE_INSENSITIVE));
```

Comment:
sorry for the lag @rhamnett.  Yeah, maxwell needs to be able to parse `PARTITION` stuff in general.  It's currently simply not supported, and is one of the last cleanups left in the sql parser section.